### PR TITLE
SW-12771 - Transfer backend components articles

### PR DESCRIPTION
**Pull request by janbuecker**
This is work in progress. Please do not merge until I tell you to do so.

### CODE REVIEW

File: source/developers-guide/backend-components/listing/index.md [hunk 1]
```diff
@@ -0,0 +1,532 @@
...
+        var date = record.get('createDate');
+        return this.booleanColumnRenderer((date.getMonth() === 6));
+    }
+});
+```
+
+In diesem Beispiel ist eine Besonderheit enthalten. Und zwar ist die Funktion `booleanColumnRenderer`. Diese Funktion wird in beiden Beispielen verwendet ohne dass diese im Shopware.grid.Panel noch im Enlight.app.Controller definiert wurde. Hierbei handelt es sich um eine Helfer Funktion von Shopware, welche sich in der `Shopware.model.Helper` <b>Klasse</b> befindet. Diese <b>Klasse</b> wird in allen Komponenten per Ext JS mixin eingebunden. Dadurch stehen sämtliche Funktionen dieser Klasse im `this` Scope zur Verfügung. Die `booleanColumnRenderer` Funktion im Main Controller Beispiel steht zur Verfügung da die Renderer Funktion als Scope das Grid Panel übergeben bekommt.
+
+
+### Plugin Download - [SwagProduct.zip](http://community.shopware.com/files/downloads/swagproduct-14067634.zip)
```
Comment (bcremer) [inline source/developers-guide/backend-components/listing/index.md]: Das Plugin bitte in das Repository unter example plugins committen. Das wird dann automatsich als zip gepackt. Siehe README.md

**Comment by bcremer** [comment]
Please remove source/developers-guide/backend-components/basics/swagproduct-201510081333.zip

**Comment by janbuecker** [comment]
Can be reviewed now.

**Comment by tiagojsag** [comment]
Somethings that need reviewing:
- Make sure the code examples (in both the plugins and the .md files) are in english, including texts/snippets
- The 'associations' guide is huge, and half of it seems to cover basic Doctrine association definition. I would remove it, point to the official Doctrine docs, and just explain what you are doing and the end result (skip the "how", unless it differs from the Doctrine official docs).
- Double check that all images have an alt value

**Pull request merged**